### ISSUE TITLE
Thanks for this great library!

### ISSUE THREAD

**Issue by supnate** [farzher/fuzzysort#30]
Not an issue but just want say thank you for this amazing project!

I thought it would be easy to find a fuzzy search lib but after I tried almost all "fuzzy" lib on npm finally find this is the only one that works as expected.

I used it in my project at: http://demo.rekit.js to support Cmd + P to quick open a file. Just for your information.

Thanks again!

**Comment by farzher** [comment]
> I thought it would be easy to find a fuzzy search lib but after I tried almost all "fuzzy" lib on npm finally find this is the only one that works as expected.

yup... same here. i was constantly surprised by how terrible they all were ^^

your project looks interesting, thanks for sharing. i want my next project to be a code editor, but it's a tough project and my ideas are still incomplete

**Comment by alinnert** [comment]
I also want to leave a biiig thank you! I'm using it on https://webdevhome.github.io/
It's basically a list of links to sites I as a web developer use very frequently. Today I've added a search to quickly focus a link (you can just type ahead to active the search).

I've also tried a few libraries and now I'm wondering: is this "letter by letter" search really part of what fuzzy search should do? Most fuzzy search libs only check if you forget or add a letter or exchange two letters etc. I almost wanted to ask what those searches are called that match "some-code-file.js" even if you enter "sooies". But on the other hand: the goto file/symbol feature in VSCode is also called "fuzzy search" in the code. Maybe the term itself is also a little... fuzzy? 😄
